app/controllers/applications_controller.rb

```diff
@@ -12,6 +12,8 @@ def index
       @authority = Authority.find_by_short_name_encoded!(params[:authority_id])
       apps = @authority.applications
       @description << " from #{@authority.full_name_and_state}"
+      @alert = Alert.new
+      @alert.address_for_placeholder = apps.last.address if apps.any?
```

Comment:
Why should this be restricted to when there is an authority. Why not always show it on the applications page? Hardly anyone hits the applications index page, but if they do the same logic about signing up from there holds I think. I'll remove the form from the conditional logic to simplify this.